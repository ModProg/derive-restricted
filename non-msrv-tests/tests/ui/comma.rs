#[derive(derive_where::DeriveWhere)]
#[derive_where(Clone,; T)]
struct Test1<T>(T);

#[derive(derive_where::DeriveWhere)]
#[derive_where(Clone,,)]
struct Test2<T>(T);

#[derive(derive_where::DeriveWhere)]
#[derive_where(Clone; T;)]
struct Test3<T>(T);

#[derive(derive_where::DeriveWhere)]
#[derive_where(Clone; T,,)]
struct Test4<T>(T);

fn main() {}
