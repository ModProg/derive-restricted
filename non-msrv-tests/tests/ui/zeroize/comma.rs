#[derive(derive_where::DeriveWhere)]
#[derive_where(,)]
struct Test1<T>(T);

#[derive(derive_where::DeriveWhere)]
#[derive_where(,Clone)]
struct Test2<T>(T);

#[derive(derive_where::DeriveWhere)]
#[derive_where(Clone,,)]
struct Test3<T>(T);

fn main() {}
