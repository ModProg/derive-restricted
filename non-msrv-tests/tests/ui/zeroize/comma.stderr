error: unsupported trait syntax, expected one of expected one of Clone, Copy, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd, Zeroize
 --> tests/ui/zeroize/comma.rs:2:16
  |
2 | #[derive_where(,)]
  |                ^

error: unsupported trait syntax, expected one of expected one of Clone, Copy, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd, Zeroize
 --> tests/ui/zeroize/comma.rs:6:16
  |
6 | #[derive_where(,Clone)]
  |                ^

error: unsupported trait syntax, expected one of expected one of Clone, Copy, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd, Zeroize
  --> tests/ui/zeroize/comma.rs:10:22
   |
10 | #[derive_where(Clone,,)]
   |                      ^
